### ISSUE TITLE
Error: Cannot find module './path.cjs' while building on next-i18next v9.0.0

### ISSUE THREAD

**Issue by artcheeze** [isaachinman/next-i18next#1518]
Error: Cannot find module './path.cjs' while building

![image](https://user-images.githubusercontent.com/25684433/141442374-4d208474-1c55-4c26-8c4e-c58f0663c733.png)

next-i18next 9.0.0 & 8.10.0 & 8.9.0
yarn 1.22.10
next 11.1.2
node 14.18.0

## I try to fix it

I've rename "path.cjs" to "path.js" at readFile.js at i18next-fs-backend package directory in node_modules then it worked! 🤔

![image](https://user-images.githubusercontent.com/25684433/141441701-cd12c42e-dc09-4f83-a2ce-ca7740ab7be1.png)

**Comment by isaachinman** [comment]
So this is not specific to v9.0.0? This looks like a module resolution issue. Can you provide a reproducible repository?

**Comment by artcheeze** [comment]
I cannot provide a reproducible repository due to confidential,
But i try to upgrade yarn version to 1.22.10 and clear yarn cache before install packages and then yarn build, now it worked on my localhost.

I will try this solution on remote server with devOps first.

**Comment by artcheeze** [comment]
FYI
![image](https://user-images.githubusercontent.com/25684433/141451001-d7aa3f8c-4363-432f-adeb-6b651bb22fd5.png)

**Comment by TheFullResolution** [comment]
So when I tested this release locally all worked, because I still had old dependencies installed. However, when deployed, our pipeline failed. 

Adding `i18next-fs-backend` as a dependency is fixing the issue. 

```
npm i i18next-fs-backend
```

**Comment by isaachinman** [comment]
@TheFullResolution You shouldn't need to add that package as a direct dependency. Can anyone provide a reproducible repo?

**Comment by e0eee** [comment]
@isaachinman
```
Module not found: Can't resolve 'fs' in '/nextjs/node_modules/next-i18next/dist/commonjs'
Native Node.js APIs are not supported in the Edge Runtime with `concurrentFeatures` enabled. Found `fs` imported.
```

**Comment by isaachinman** [comment]
@e0eee That seems like you're trying to use `next-i18next` in NextJs middleware, and would be a separate issue.

**Comment by TheFullResolution** [comment]
@isaachinman I think the problem solved itself - https://github.com/i18next/i18next-fs-backend/issues/25

**Comment by TheFullResolution** [comment]
I can confirm all works now, including our pipelines...

**Comment by alipiry** [comment]
Hi @isaachinman, I am using NextJS v12.0.1 alongside next-i18next v9.0.0 and I'm facing this issue. Any fixes on it? 

<img width="1437" alt="image" src="https://user-images.githubusercontent.com/25138854/141657287-f65b1d88-f031-4176-b09c-eec8d1cf9658.png">

**Comment by isaachinman** [comment]
Fix released in v9.0.1. If anyone has any further issues, let me know.
